REMOVE_THIS.md

```diff
@@ -0,0 +1 @@
+REMOVE THIS
```

Comment:
There is literally no need for this file if this PR also removes actual code

Reply:
The PR only contained this file initially, I didn't realize I needed other code changes until later.

Reply:
ah. worth taking this particular file back out then?